Testscripts/Linux/ovs_setup.sh

```diff
@@ -79,7 +79,7 @@ function install_ovs () {
 	ssh "${1}" "cd ${OVS_DIR} && ./boot.sh"
 
 	LogMsg "Starting OVS configure on ${1}"
-	ssh "${1}" "cd ${OVS_DIR} && ./configure --with-dpdk=${RTE_SDK}/${RTE_TARGET} --prefix=/usr --localstatedir=/var --sysconfdir=/etc"
+	ssh "${1}" "cd ${OVS_DIR} && ./configure --with-dpdk=static"
```

Comment:
Could you please add comments here for difference of "static" and previous setting?

Reply:
I followed guidelines to install OVS from manually installing DPDK.